appveyor.yml

```diff
@@ -28,18 +28,18 @@ install:
   - set PATH=C:\Program Files (x86)\NSIS;%PATH%
   - set PATH=C:\MinGW\bin;%PATH%
   - set PATH=C:\MinGW\msys\1.0\bin;%PATH%
-  - .\scripts\ci\appveyor-install.bat
+  - bash .\scripts\ci\install.sh -o win32 -r %TARGET_ARCH%
 
 build: off
 
 test_script:
   - node --version
   - npm --version
-  - .\scripts\ci\appveyor-test.bat
-  - .\scripts\ci\appveyor-build-installers.bat
+  - bash .\scripts\ci\test.sh -o win32 -r %TARGET_ARCH%
+  - bash .\scripts\ci\appveyor-build-installers.sh -o win32 -r %TARGET_ARCH%
```

Comment:
The CI build has just failed - looks like somebody forgot to remove the word `appveyor-` from this line ;-)